tests/aws/services/cloudformation/resources/test_lambda.py

```diff
@@ -487,9 +487,22 @@ def test_multiple_lambda_permissions_for_singlefn(deploy_cfn_template, snapshot,
 
 
 @markers.aws.validated
-def test_lambda_function_tags(deploy_cfn_template, aws_client):
+@markers.snapshot.skip_snapshot_verify(
+    paths=[
+        # Added by CloudFormation
+        "$..Tags.'aws:cloudformation:logical-id'",
+        "$..Tags.'aws:cloudformation:stack-id'",
+        "$..Tags.'aws:cloudformation:stack-name'",
+    ]
+)
+def test_lambda_function_tags(deploy_cfn_template, aws_client, snapshot):
+    snapshot.add_transformer(snapshot.transform.lambda_api())
+    snapshot.add_transformer(snapshot.transform.key_value("CodeSha256"))
+
     function_name = f"fn-{short_uid()}"
     environment = f"dev-{short_uid()}"
+    snapshot.add_transformer(snapshot.transform.regex(environment, "<environment>"))
```

Comment:
q: is this needed? Why can't this be a regular `key_value` transformer with the `Environment` key?

Reply:
When the value is specific here (like a randomly generated ID that's not a standard part of the API) I like my transformations to be specific. Just a personal preference.

Reply:
I see. I'm not too fond of the `regex` transformer myself, as it acts on the raw JSON string, but I can see the point.